lib/client.js

```diff
@@ -0,0 +1,177 @@
+const ec = require("elliptic");
+const { rnodeDeploy, signDeploy } = require("@tgrospic/rnode-grpc-js");
+const protoSchema = require("./rnode-grpc-gen/js/pbjs_generated.json");
+const grpc = require("@grpc/grpc-js");
+require("../lib/rnode-grpc-gen/js/DeployServiceV1_pb");
```

Comment:
This is not necessary because proto generated JS is initialized in `./lib/proto.js` file and loaded in `index.js`.